src/ui/components/results/verticalresultscomponent.js

```diff
@@ -254,9 +256,13 @@ export default class VerticalResultsComponent extends Component {
     };
   }
 
-  mount () {
+  mount (container) {
```

Comment:
could this ben done in an onMount instead? same question for universal results, it's just a little more common to do this kind of stuff in the onMount instead of overriding mount